frontend/admin/webpack.base.js

```diff
@@ -99,6 +100,12 @@ module.exports = {
       },
     ],
   },
+  optimization: {
```

Comment:
Does this take a noticable amount of time to run?  This seems like something that should only be in the PROD webpack config.  It may get in the way of troubleshooting, too.

Reply:
Hmm, you're right it only runs in production even thought the config is in the base file.  Is there a way to make a default behavior explicit so that it's clearer?  Or at least a comment to flag non-obvious behavior?  Regarding speed, prod build is already quite a bit slower and I'm not noticing any further slowdown.

Reply:
Sounds good, I added a comment with [add note that optim runs only in prod mode](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3447/commits/d7974d93ddb09d2a73e2ae0b04414d718ed6165b). Let me know if it needs some additional information.